src/v2/components/ChannelContents/index.tsx

```diff
@@ -131,7 +131,7 @@ const ChannelContents: React.FC<Props> = WithIsSpiderRequesting<ExtendedProps>(
 
     usePusher({
       channelId: channel.id,
-      shouldSubscribe: !isSpiderRequesting && channel.can.add_to,
+      shouldSubscribe: !isSpiderRequesting && !!channel?.can?.add_to,
```

Comment:
A lot of these null-checking changes are because I had strictNullChecks on in my local branch while working on the table. I didn't want to go overboard with upgrading our entire codebase, but I made some changes to files that I consistently had open to increase clarity.